### DISCUSSION TITLE
Tailwind 3.0.0-alpha release

### DISCUSSION THREAD

**Discussion by jeffreyvr** [tailpress/tailpress#65] [General]
For those wanting to test with [Tailwind v3.0.0-alpha.1](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0-alpha.1), check out this branch:

https://github.com/jeffreyvr/tailpress/tree/mix-tailwind-3

This branch uses Laravel Mix again. 

Only needed to make some minor adjustments. Haven't found issues so far.

**Comment by tdecoy** [comment]
How do I use it? Sorry, new to Git

**Reply by jeffreyvr** [reply]
You can clone the repository and then checkout the `mix-tailwind-3` branch.

This should work when you run it in your terminal:
```bash
git clone https://github.com/jeffreyvr/tailpress.git && cd tailpress && git checkout mix-tailwind-3
```

**Reply by tdecoy** [reply]
That works, thanks Jeff.

Couple of follow-ups:

1) is the Getting Started guide outdated? I don't see a .git folder anywhere.

2) how do I make it work with browser-sync? Is there a port it opens up somewhere when running npm run watch? Having to manually refresh the browser between saves is a bit of a pain.

3) how do I update packages? Nothing responds to npm update, not even tailwindcss (when using main branch, that is)

Thanks man, appreciate the help and sorry for the noob questions.

**Reply by tdecoy** [reply]
4. when trying to install @tailwindcss/typography it complains about only being for Tailwindcss v2. Does it just not work in this alpha or is there some trickery I'm missing?

**Reply by jeffreyvr** [reply]
Regarding point 4: check out https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0-alpha.1

**Reply by tdecoy** [reply]
BUMP 👍

**Reply by jeffreyvr** [reply]
Side note: Most of the questions are not specifically related to Tailwind CSS 3.0.0, so it's a little off topic. 

1. Getting started is not getting outdated, but the .git folder might not be visible due to settings on your computer. If you clone the repo, that folder must be there.

2. For browser-sync, make sure you adjust the site-URL here: https://github.com/jeffreyvr/tailpress/blob/mix-tailwind-3/package.json#L20

3. Hm, that's strange. I don't know why that is to be honest.

**Reply by tdecoy** [reply]
Thanks for that, Jeff. Appreciate it

**Reply by tdecoy** [reply]
Hey Jeff

Don't know if it matters, but here's a quick update:

Couldn't get `watch-sync` to work with the current git. On a fresh clone/install it kept saying:

> 'concurrently' is not recognized as an internal or external command,
> operable program or batch file.

`npm update` did nothing. Had to remove `concurrently` and reinstall, which then installed the latest version.

Now it works, and localhost:3000 popped up correctly.

Probably doing something wrong, but there you go.
